shell/src/main/java/alluxio/cli/fs/command/LsCommand.java

```diff
@@ -238,7 +240,19 @@ private void ls(AlluxioURI path, boolean recursive, boolean forceLoadMetadata, b
       optionsBuilder.setLoadMetadataType(LoadMetadataPType.ALWAYS);
     }
     optionsBuilder.setRecursive(recursive);
+
+    // If list status takes too long, print the message
+    Timer timer = new Timer();
+    timer.schedule(new TimerTask() {
```

Comment:
nit: use Lambda Expression? https://docs.oracle.com/javase/tutorial/java/javaOO/lambdaexpressions.html

Reply:
I tried lambda expression and itellij complains that `cannot resolve method...`. According to https://stackoverflow.com/questions/37970682/passing-lambda-to-a-timer-instead-of-timertask, TimerTask is not a SAM (single abstract method) type and cannot directly use lambda to replace the TimerTask. Thanks!